modules/seastate/src/SeaState.txt

```diff
@@ -89,8 +91,11 @@ typedef   ^                  InitOutputType         CHARACTER(ChanLen)       Wri
 typedef   ^                  ^                      CHARACTER(ChanLen)       WriteOutputUnt    {:}     -      -  "The is the list of all HD-related output channel unit strings (includes all sub-module channels)"    -
 typedef   ^                  ^                      ProgDesc                 Ver                -      -      -  "Version of SeaState"
 typedef   ^                  ^                      LOGICAL                  InvalidWithSSExctn -      -      -  "Whether SeaState configuration is invalid with HydroDyn's state-space excitation (ExctnMod=2)" (-)
-typedef   ^                  ^                      SiKi                     WaveElevSeries    {:}{:}  -      -  "Wave elevation time-series at each of the points given by WaveElevXY.  First dimension is the timestep. Second dimension is XY point number corresponding to second dimension of WaveElevXY." (m)
+typedef   ^                  ^                      SiKi                     WaveElevVisX       {:}     -      -  "X locations of grid output"    "m,-"
+typedef   ^                  ^                      SiKi                     WaveElevVisY       {:}     -      -  "X locations of grid output"    "m,-"
```

Comment:
`WaveElevVisY` should say `Y locations of grid output`, not `X` (was done correctly in FAST_Registry.txt).

Reply:
Good catch!